build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
Rama I'm not sure I get the goal behind this suggestion just yet. Presumably if devs wanted to build on x64 and copy the tests to ARM/ARM64 they would run build-test.cmd on windows, then copy all of the output to the arm machine. After this change, that set of binaries would now include debugger tests. It seems fairly straightforward but perhaps there is something I am missing?

As for consistency, I agree that it is inconsistent that runtest.sh builds tests and runtests.cmd does not, but my understanding is that is a pre-existing inconsistency which applies to all tests? If only the debugger tests broke from that convention and instead always downloaded/built the tests in runtests.*, we would be consistent across OS's but now inconsistent from how all the other tests work. Its not clear to me that trading one consistency for the other would be an improvement. Consistency aside, it seems valuable from a perf perspective to be able to download and build all the tests once so that you don't pay the perf costs repeatedly. I also find the model that running tests doesn't have network dependencies to be a pretty useful property. I generally strive to isolate network dependencies as much as possible and put them as early in the automation pipeline as possible so that if they fail, it fails at a point where a human is hopefully still paying attention and can take remedial action. Failing a network request 20 minutes in to 30 minute automated process is usually really aggravating.